data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+		random < 45
+	source
+		attributes arach
+	destination "Chosen Nexus"
+	on offer
+		conversation
+			`An elderly Arach beckons to you as you enter the spaceport. He is accompanied by four Arachi children, who all stand close to a large pile of luggage.`
+			`	He explains that he used to run an orphanage here, but has now gotten too old for the job. He has arranged for the four kids to be transferred to a foster home on <destination>.`
+			`	"Take care of them, the Saryds there will," he says, patting the head of the youngest and shortest of the kids. "Retiring, I am, but abandon them, I will not. An apartment close to the foster home, I will purchase."`
+			`	He offers to pay you <payment> to bring the five of them and their <cargo> to <destination>.`
+			choice
+				`	"Of course I'll help. Here, let me help you get the luggage into my ship."`
+					accept
+				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`
+					decline
+	on complete
+		"coalition jobs" += 3
+		payment 268000
+		conversation
+			`You help the old Arach caretaker and some workers take the luggage out of your ship then head to the orphanage.`
+			`	A Saryd couple runs the place and greets you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child and some Kimek too.`
+			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
+
+
+
+mission "Arachi Cartoonist 1"
+	minor
+	name "A New Design"
+	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
+	to offer
+		has "Coalition: First Contact: done"
+	source "Mebla's Portion"
+	destination "Prime"
+	on offer
+		conversation
+			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you in your tracks.`
+			`	He introduces himself as Drugub, a cartoonist of local fame for his sci-fi comics. "Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
+			choice
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
+					decline
+			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
+			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs and settles for those.`
+			`	He thanks you and wishes you safe travels on your journey to <planet>.`
+				accept
```

Comment:
Got bored. Wrote this.
```suggestion
			`As you're moving along one of the underground tunnels that connect the spaceport here, you notice an Arach off in the distance looking at you. He shouts something in Arachi, then begins running toward you while fiddling with a translation box.`
			`	By the time he reaches you he is moving much slower than when he started. "Vistitor, forgive me," he says, breathing heavily while resting against the nearest wall. "Miss this opportunity to meet you, I could not."`
			`	The Arach takes a deep breath and stands upright. "Drugub, I am. A creator of science fiction comics, I am famous for. Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would. Travel to human space, you may, and return with images for inspirations?"`
			choice
				`	"I can help with that. Do you have any type of ship in mind?"`
				`	"Sorry, but I won't be heading back to human space for a while."`
					decline
			`	Drugub lets out a noise that you would assume is a laugh. "Of the ships of your people, I know not. Please describe them."`
			`	You describe the various shipyards of human space; Lionheart, who develop some of the most advanced ships in human space; Tarazed, who design their ships with inspiration from the Quarg (resulting in a noticeable grimace from Drugub); Syndicate, who specialize in cheap yet (somewhat) effective ships.`
			`	After describing the ships of Betelgeuse Shipyards on <planet> and how some of their designs are almost as old as humanity's hyperdrive, he stops you. "Perfect, this would be. Set during a... less developed time, my series is. Older designs, I have been searching for."`
			choice
				`	"Okay. I'll look for something to bring back for you."`
					goto end
				`	"Sounds like a plan. Let's get ready to leave."`
			`	"Us?" Drugub asks, letting out another noise that you think is a laugh. "No, only you may go. Leave Coalition space, the Heliarchs do not allow. Permission, I require. Without me, you must go. Bring images for me is all I need."`
			label end
			`	He thanks you and wishes you safe travels on your journey to <destination>.`
				accept
```